events/story_cycles/wc_story_cycle_horde_invasion_events.txt

```diff
@@ -47,39 +47,109 @@ WCHOI.2 = {
 	}
 	right_portrait = scope:stormwind_holder
 
-	option = { # Corrupted orcs option
-		trigger = {
-			NOT = { 
-				faith = faith:orcish_shamanism
+	option = {
+		name = {
+			text = {
+				first_valid = {
+					triggered_desc = { # Corrupted orcs option
+						trigger = { faith = faith:orcish_fel }
+						desc = WCHOI_2_OPT_A
+					}
+					triggered_desc = { # Frostwolf option
+						trigger = { faith = faith:orcish_shamanism }
+						desc = WCHOI_2_OPT_B
+					}
+					desc = WCHOI_2_OPT_C # Outsider option
+				}
 			}
-			faith = faith:orcish_fel
 		}
-		name = WCHOI_2_OPT_A
 	}
+}
+
+# Ends the Horde Invasion when enough lands taken
+WCHOI.3 = { # Save the Horde?
+	type = character_event
+	title = WCHOI_2_TITLE
+	desc = WCHOI_3_DESC
+	theme = war
+	override_background = {
+		reference = wc_background_dark_portal
+	}
+	left_portrait = root
 
-	option = { # Frostwolf option
+	option = { # The orcs need their warchief!
 		trigger = {
-			NOT = { 
-				faith = faith:orcish_fel
+			custom_description = {
+				text = can_save_the_first_horde_tooltip
+				OR = {
+					has_game_rule = wc_collapse_of_the_first_horde_never
+
+					NOT = {
+						pol_faction_exists_trigger = { faction = alliance } # TODO - update this when we add dismantle cb for all unions
```

Comment:
So thinking about this, it doesn't make much sense. The orcs united under one warchief because there was a threat and they wanted to expand to new lands. So if there's no Alliance, there's definitely no reason to keep the Horde.

The only reason that may stay united is the Warchief breaking their agreement and going a full tyrant. I don't see such outcomes here. The Warchief gets prestige instead.

Reply:
The previous idea of giving the modifier to suppress other clans looked better.